scss/modules/_typography-fonts.scss

```diff
@@ -25,3 +25,6 @@ $headingFont: 'Roboto Slab', 'serif';
  * }
  */
 $mainFont: 'Helvetica Neue', 'Helvetica', Arial, sans-serif;
+
+@import url("http://fonts.googleapis.com/css?family=Open+Sans:800");
+$headingEventFont: 'Open Sans', 'sans-serif';
```

Comment:
As I see, other projects in style guide don't use Open Sans. 
So should I make Open Sans default font for headings? Or should I remove it and use Roboto, as it was before?